glmmTMB/src/glmmTMB.cpp

```diff
@@ -579,6 +579,11 @@ Type allterms_nll(vector<Type> &u, vector<Type> theta,
 template<class Type>
 Type objective_function<Type>::operator() ()
 {
+#ifdef _OPENMP
```

Comment:
I will assume that omp_get_max_threads() will always pull in the specified threads instead of every thread in the system right? (It's been a while since I've written R / C++)

Reply:
not sure ...

Reply:
Yes, `omp_get_max_threads()` pulls the specified number of threads - *not* the system max.